Rider/app/src/main/res/drawable/forward_arrow.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#4A4A4A"
+    android:viewportHeight="24.0" android:viewportWidth="24.0"
```

Comment:
Understand: Is there any reason you aren't using dp on this? Not saying you should or should not, just want to double check.